build.sh

```diff
@@ -125,6 +127,10 @@ QUALIFIED_KERNEL="$(rpm -qa | grep -P 'kernel-(|'"$KERNEL_SUFFIX"'-)(\d+\.\d+\.\
 # We could get some kind of static binary for GCC but this is the cleanest and most tested alternative. This Sucks.
 dnf -y --setopt=install_weak_deps=False install gcc
 
+# The compose repos we used during the build are point in time repos that are
+# not updated, so we don't want to leave them enabled.
+dnf config-manager --set-disabled baseos-compose,appstream-compose
+
```

Comment:
Can I ask why we shouldn't leave them enabled?
From what I understood, downstream builds (i.e. anyone building off this image) is also broken unless they do the same workarounds as us.  And dnf is not functional on booted ostree systems